### PR TITLE
Done task 5

### PR DESCRIPTION
**Pull request by agnieszka-pytel**

### CODE REVIEW

File: task5/task5.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                    Nulla posuere, nibh non dignissim rutrum, massa magna posuere nunc, eget accumsan ante mi quis velit. Curabitur porta nisl at odio tincidunt, eget accumsan erat ornare.
+                </p>
+                <p>
+                    Mauris ut eros nulla. Nunc at dapibus erat, id cursus risus. Sed rutrum, lectus tempor scelerisque ultrices, eros dui tincidunt arcu, ut egestas metus justo non neque. Quisque euismod sagittis dictum. Fusce varius, elit in rhoncus venenatis, ligula lorem
+                    vehicula odio, id pharetra est ligula eu sapien. Nunc mollis dolor suscipit ligula sagittis interdum.
+                    <blockquote class="quote">Donec consectetur risus sed mauris sagittis, id bibendum est bibendum.</blockquote>
+                    Ut vitae pretium sapien, nec ultricies nunc. Aliquam erat volutpat. Cras et imperdiet risus, at dapibus tellus. Phasellus augue nisi, tincidunt non mi et, sodales hendrerit orci. Nam suscipit placerat diam in sodales. Suspendisse ut tristique mauris.
```
Comment (Tsvetkovv) [inline task5/task5.html]: wrap it as well please. Don't left text without an element

File: task5/task5.css [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    opacity: 0.5;
+}
+
+#sans-serif:checked+.sans-serif,
+#serif:checked+.serif,
+#small-font:checked+.small-font,
+#normal-font:checked+.normal-font,
+#large-font:checked+.large-font,
+#day-scheme:checked+.day-scheme,
+#night-scheme:checked+.night-scheme {
```
Comment (Tsvetkovv) [inline task5/task5.css]: please set single css class for that to avoid enumeration 7 selectors

Comment (agnieszka-pytel) [reply task5/task5.css]: But I want every pair to change style independently

Comment (Tsvetkovv) [reply task5/task5.css]: sure, you may add second class

Comment (Tsvetkovv) [reply task5/task5.css]: this rule is specific not only for sans-serif, serif, small-font etc.
this rule is for selected elements and their label.

Comment (agnieszka-pytel) [reply task5/task5.css]: Ok, I got it now, I underestimated the `+` operator

File: task5/task5.css [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+.serif,
+#serif:checked~.content-wrapper {
+    font-family: 'Times New Roman', Georgia, Times, serif;
+}
+
+.small-font,
+#small-font:checked~.content-wrapper {
+    font-size: 0.85rem;
```
Comment (Tsvetkovv) [inline task5/task5.css]: just to be sure, do you understand that with pixels here you would have the same effect?

Comment (agnieszka-pytel) [reply task5/task5.css]: Yes, except that using rems allows me to change only one line of css (root font-size) when I would like to change overall layout and everything would scale automatically

**Pull request merged**